Description: PAN GlobalProtect: pass "gp" protocol to openconnect
 OpenConnect v8.00 (1) introduced support for PAN GlobalProtect VPN protocol with the --protocol=gp option.
 [1]: http://www.infradead.org/openconnect/changelog.html
Author: Sergi Granell <xerpi.g.12@gmail.com>
Origin: upstream, https://gitlab.gnome.org/GNOME/NetworkManager-openconnect/commit/0a1bc7a7a2c5e8811cb5113d0ec6effc0024daa2
Bug: https://gitlab.gnome.org/GNOME/NetworkManager-openconnect/issues/1
Bug-Debian: https://bugs.debian.org/925081
Last-Update: 2019-05-05
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/properties/nm-openconnect-editor-plugin.c
+++ b/properties/nm-openconnect-editor-plugin.c
@@ -407,6 +407,9 @@
 		} else if (nm_streq (add_detail, "nc")) {
 			NM_SET_OUT (out_pretty_name, g_strdup (_("Juniper Network Connect (openconnect)")));
 			NM_SET_OUT (out_description, g_strdup (_("Compatible with Juniper Network Connect / Pulse Secure SSL VPN")));
+		} else if (nm_streq (add_detail, "gp")) {
+			NM_SET_OUT (out_pretty_name, g_strdup (_("PAN GlobalProtect (openconnect)")));
+			NM_SET_OUT (out_description, g_strdup (_("Compatible with PAN GlobalProtect SSL VPN")));
 		} else {
 			/* we don't know this protocol by name, but it's configured in the .name file,
 			 * so just take it. */
@@ -524,6 +527,7 @@
 	char *dflt[] = {
 		"anyconnect",
 		"nc",
+		"gp",
 		NULL,
 	};
 
